jax/_src/numpy/reductions.py

```diff
@@ -23,12 +23,15 @@
 
 import numpy as np
 
+import jax
+from jax import lax
+from jax import numpy as jnp
```

Comment:
Unfortunately, we can't import top-level submodules here because it creates circular dependencies. Please only import from `jax._src`.